mergeable/src/lib/vendor.d.ts

```diff
@@ -0,0 +1,26 @@
+declare namespace GitHubApi {
```

Comment:
this is for the `github-api` library https://www.npmjs.com/package/github-api to be honest I had a whole bunch of trouble trying to declare types for it